lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -402,37 +461,51 @@ def _quoted_partman_schema
   end
 
   def _fully_qualified_table_name_for_partman(table)
+    _schema_and_table_for(table).each do |identifier|
+      if identifier.to_s !~ /^[a-z_][a-z_\d]*$/
+        raise PgHaMigrations::InvalidMigrationError, "Partman requires schema / table names to be lowercase with underscores"
```

Comment:
Could this include the invalid name in its output? And if we do that, we probably need to not short-circuit (i.e., we want to know if _both_ parts are invalid).

Reply:
Yeah I can look into that

Reply:
If you're keeping track of that, that's good for me.